mgr/.pawlogon.kumac

```diff
@@ -1,6 +0,0 @@
-kuip/alias/create opt  'graphics/opt'  c
```

Comment:
Well, this may or may not be our standard logon.  I *think* this gets copied into asps/ during the build process, but am not 100% certain.  Regardless, this is not a ROOT6 relevant modification and should not be part of the PR.